introduction/deprecated_features/CAPMC_Deprecation_Notice.md

```diff
@@ -1,47 +1,41 @@
 # CAPMC Deprecation Notice
 
-## CAPMC enters final life cycle before deletion
+## CAPMC enters final life cycle before removal
 
-PCS v1 has been released to production and indicated the beginning of the final life cycle for CAPMC. CAPMC will
-continue to co-exist with PCS until its final deletion. CAPMC is slated to be permanently removed from the system
-beginning with the first release of software in 2024. Please begin to migrate to PCS.
+CSM 1.5 is the last release that will contain the CAPMC service. It has been
```

Comment:
This seems inaccurate if CAPMC is still in CSM 1.6, but just not supported. Should this paragraph be reworded?

Reply:
Ooops, that was the 1.5 verbiage.  Should be removed and replaced with what I added.  Will update.

Reply:
I still see this saying that CAPMC is removed in 1.6. Did you push up your changes?